sidecar-injector/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.11-stretch as mutating-webhook-service-builder
+FROM golang:latest as mutating-webhook-service-builder
 
 ENV GO111MODULE=on
```

Comment:
taking it out

Reply:
turns out this is necessary, not sure why but taking it out resulted in complaints about glog

Reply:
hmm..that's pretty odd but fair enough. Can you just add a TODO note that we should figure out why we can't take that out (since we really shouldn't need it)